docs/guides/Payments/payments-configuration-guides/implementing-a-pre-analysis-antifraud-flow-for-debit-card-transactions.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Implementing a pre-analysis antifraud flow for debit card transactions"
+slug: "implementing-a-pre-analysis-antifraud-flow-for-debit-card-transactions"
+hidden: false
+createdAt: "2023-07-04T00:00:00.691Z"
+updatedAt: ""
+---
+
+To provide more protection in transactions processed by debit cards, an initial pre-analysis antifraud step can be added to the standard payment transaction flow.
```

Comment:
No [glossário de Localization](https://docs.google.com/spreadsheets/d/1vSSg5CrimqSxvWjBoXLE5P7ATJCdRnPT/edit#gid=1978438735&range=A14), parece que a escolha é utilizar `anti-fraud solution` em inglês. Recomendo enviar esse PR para revisão delas para darem uma conferida nesses termos. Se tiver alguma dúvida sobre esse processo pode chamar a @camilavcoutinho.

Reply:
Neste caso, acho que não vai fazer sentido. O próprio pessoal de localization utilizou um link de referência em um contexto onde é descrito como _anti-fraud_ apenas. (https://help.vtex.com/en/tutorial/what-is-anti-fraud--69SjFCc4rC6Ii0OMAeYAsG)